master/buildbot/test/unit/test_reporter_gerrit.py

```diff
@@ -463,3 +464,22 @@ def testBuildGerritCommand(self):
             'ssh',
             ['ssh', 'user@serv', '-p', '29418', 'gerrit', 'review', '--project project',
              "--message 'bla'", '--verified 1', 'revision'])
+
+        # now test the notify argument
+        gsp.gerrit_notify = 'OWNER'
+        gsp.processVersion("2.6", lambda: None)
```

Comment:
Well, per [String Quotes](https://www.python.org/dev/peps/pep-0008/#string-quotes) in PEP-8, both `"` and `'` are OK, however it's good to stick to some particular way.

So far, we've been using `"` for docstrings only `"""` and `'` for all other string literals (some legacy code does not follow this, I must admit).